backend/experiment/actions/final.py

```diff
@@ -122,6 +125,7 @@ def __init__(
         self.show_profile_link = show_profile_link
         self.show_participant_link = show_participant_link
         self.show_participant_id_only = show_participant_id_only
+        self.show_percentile = show_percentile
         self.feedback_info = feedback_info
         self.logo = logo
         self.percentile = session.percentile_rank(exclude_unfinished=False)
```

Comment:
IMO it's not even necessary to pass the `show_percentile` boolean to the frontend. We could set `self.percentile` to None if this is `False`, right? Or are there scenarios in which we don't want to show the percentile, but still pass the value?